client/src/components/products/ProductsList.js

```diff
@@ -87,13 +88,27 @@ class ProductsList extends Component {
       )
     ];
 
+
+    const distinctCity = [
+      ...new Set(
+        this.state.products.map(product => {
+          return product.location.city;
+        })
+      )
+    ];
+
+    
+
     // const maxPrice = Math.max(
     //   ...this.state.products.map(product => {
     //     return product.price;
     //   })
     // );
```

Comment:
Do we still need this? If not, let's remove it?

Reply:
done! 💨